packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2362,6 +2362,12 @@ class InputDecoration {
 
   /// The shape of the border to draw around the decoration's container.
   ///
+  /// All border side information contained in this [border] will be overriden
```

Comment:
This border's [InputBorder.borderSide], i.e. the border's color and width, will be overridden to reflect the input decorator's state. Only the border's shape is used. If custom ...